Dockerfile

```diff
@@ -32,6 +32,14 @@ RUN cd /opt/ \
   && wget https://archive.apache.org/dist/spark/spark-2.3.0/spark-2.3.0-bin-hadoop2.6.tgz \
   && tar -xvf ./spark-2.3.0-bin-hadoop2.6.tgz \
   && rm -rf /opt/spark-2.3.0-bin-hadoop2.6.tgz
+RUN cd /opt/ \
+  && wget https://www.python.org/ftp/python/3.6.3/Python-3.6.3.tgz \
```

Comment:
isn't it easier to install it with `sudo apt-get install python3.6`?

Reply:
it uses ubuntu 16.04 as a base which doesn't have python 3.6 available

Reply:
👍